jadx-gui/src/main/java/jadx/gui/ui/MainWindow.java

```diff
@@ -1062,11 +1073,10 @@ private void initMenuAndToolbar() {
 		flatPkgMenuItem = new JCheckBoxMenuItem(NLS.str("menu.flatten"), Icons.FLAT_PKG);
 		flatPkgMenuItem.setState(isFlattenPackage);
 
-		JCheckBoxMenuItem enablePreviewTabMenuItem = new JCheckBoxMenuItem(NLS.str("menu.enable_preview_tab"));
-		enablePreviewTabMenuItem.setState(settings.isEnablePreviewTab());
-		enablePreviewTabMenuItem.addActionListener(event -> {
+		JadxGuiAction enablePreviewTabAction = new JadxGuiAction(ActionModel.PREVIEW_TAB, () -> {
 			settings.setEnablePreviewTab(!settings.isEnablePreviewTab());
 		});
+		enablePreviewTabAction.setSelected(!settings.isDockQuickTabs());
```

Comment:
```suggestion
		enablePreviewTabAction.setSelected(settings.isEnablePreviewTab());
```

Reply:
lol, how awkward it turned out, copy paste  🤣